index.html

```diff
@@ -3,15 +3,130 @@
+                <circle cx="147.253" cy="147.253" r="104.151" transform="rotate(-43.6852 147.253 147.253)" fill="url(#paint0_linear_139_1091)" />
+                <path d="M213.451 145.8C213.451 185.541 181.235 217.758 141.493 217.758C101.752 217.758 69.5356 185.541 69.5356 145.8C69.5356 106.059 101.752 73.8422 141.493 73.8422C181.235 73.8422 213.451 106.059 213.451 145.8ZM95.9777 145.8C95.9777 170.938 116.356 191.316 141.493 191.316C166.631 191.316 187.009 170.938 187.009 145.8C187.009 120.662 166.631 100.284 141.493 100.284C116.356 100.284 95.9777 120.662 95.9777 145.8Z" fill="#E4E4E4" />
+                <path d="M141.909 62.9342C155.158 62.9866 168.202 66.2148 179.946 72.3482C191.69 78.4816 201.793 87.3414 209.407 98.1847C217.02 109.028 221.923 121.539 223.705 134.668C225.486 147.797 224.093 161.162 219.643 173.641C215.194 186.121 207.817 197.352 198.131 206.393C188.446 215.434 176.734 222.02 163.977 225.601C151.221 229.181 137.792 229.651 124.816 226.971C111.841 224.29 99.6971 218.538 89.4034 210.196L113.037 181.033C118.668 185.597 125.311 188.744 132.41 190.21C139.508 191.676 146.855 191.419 153.833 189.461C160.812 187.502 167.219 183.898 172.518 178.953C177.817 174.007 181.853 167.863 184.287 161.035C186.721 154.208 187.483 146.896 186.508 139.714C185.534 132.531 182.852 125.687 178.687 119.755C174.521 113.823 168.995 108.976 162.57 105.621C156.145 102.265 149.009 100.499 141.761 100.471L141.909 62.9342Z" fill="#6941C6" />
+                <path d="M241.837 146.275C241.745 169.548 233.575 192.067 218.723 209.985C203.871 227.903 183.259 240.109 160.407 244.516L150.1 191.08C160.572 189.06 170.018 183.467 176.824 175.256C183.63 167.045 187.374 156.725 187.416 146.06L241.837 146.275Z" fill="#42307D" />
+                <defs>
+                  <linearGradient id="paint0_linear_139_1091" x1="211.818" y1="207.105" x2="94.9421" y2="54.4126" gradientUnits="userSpaceOnUse">
+                    <stop stop-color="white" />
+                    <stop offset="1" stop-color="#F5F5F5" />
+                  </linearGradient>
+                </defs>
+              </svg>
+              <p class="Earnings__number" aria-label="The percentage of profit for the month is 40%">40%</p>
+            </div>
+          </div>
+          <dl>
+            <dt class="Earnings__decription">Speady, Easy and Fast</dt>
+            <dd class="Earnings__text">Overpay help you set saving goals, earn cash back offers, Go to disclaimer for more details and get paychecks up to two days early. Get a $20 bonus when you receive qualifying direct deposits</dd>
+          </dl>
+        </li>
+        <div class="carousel__indicators">
+          <div class="indicator js-indicator" aria-hidden="true"></div>
+          <div class="indicator js-indicator" aria-hidden="true"></div>
+          <div class="indicator js-indicator" aria-hidden="true"></div>
```

Comment:
Repeat with me: `los div no son elementos interactivos, debo usar un elemento interactivo` 😆 
```suggestion
          <button type="button" class="indicator js-indicator" aria-hidden="true"></button>
          <button type="button" class="indicator js-indicator" aria-hidden="true"></button>
          <button type="button" class="indicator js-indicator" aria-hidden="true"></button>
```